osf/models/base.py

```diff
@@ -108,6 +93,10 @@ class BaseModel(models.Model):
     class Meta:
         abstract = True
 
+    def to_storage(self):
```

Comment:
It's used in tests. I can find another way to do a logical comparison but this seemed like an easy solution that's also backwards compatible with modm.